sound/soc/sof/topology.c

```diff
@@ -1594,6 +1597,17 @@ static int sof_widget_load_pga(struct snd_soc_component *scomp, int index,
 
 	swidget->private = volume;
 
+	list_for_each_entry(scontrol, &sdev->kcontrol_list, list) {
+		if (scontrol->comp_id == swidget->comp_id) {
+			volume->min_value =
+				scontrol->volume_table[scontrol->min];
```

Comment:
so scontrol->min and scontrol->max are in what units? dB, steps?

Reply:
@plbossart it is in step, and from the struct:
`struct snd_soc_tplg_mixer_control {`
`	struct snd_soc_tplg_ctl_hdr hdr;`
`	__le32 size;	/* in bytes of this structure */`
`	__le32 min;`
`	__le32 max;`
`	__le32 platform_max;`
`	__le32 invert;`
`	__le32 num_channels;`
`	struct snd_soc_tplg_channel channel[SND_SOC_TPLG_MAX_CHAN];`
`	struct snd_soc_tplg_private priv;`
`} __attribute__((packed));`

OK, I will change to min_step, and max_step.